backend/chat/models.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from config.settings import CHOICE_ROOM
+from django.contrib.auth import get_user_model
+from django.db import models
+from PIL import Image
+
+
+class Chat(models.Model):
+    """Chat table"""
+
+    title = models.CharField(max_length=70)
+    room = models.CharField(choices=CHOICE_ROOM, max_length=50)
+    img = models.ImageField(
+        null=True,
+        blank=True,
+        default="default-user-avatar.jpg",
+        upload_to="chat_img",
+    )
+    creator = models.ForeignKey(get_user_model(), null=True, on_delete=models.CASCADE)
```

Comment:
Why do you use `CASCADE` here?

Reply:
If we delete the user, then the chats created by him are also deleted.
Do chats of deleted users have to stay?

Reply:
I changed it